lib/ansible/modules/notification/typetalk.py

```diff
@@ -19,7 +19,7 @@
 version_added: "1.6"
 short_description: Send a message to typetalk
 description:
-  - Send a message to typetalk using typetalk API ( http://developers.typetalk.in/ )
+  - Send a message to typetalk using typetalk API ( http://developers.typetalk.com/ )
```

Comment:
`http://developers.typetalk.com/` does NOT work for me. `http://developers.typetalk.in` works fine.